TFTP macOS instructions

evanreichard opened the pull request:
Hey! I recently went through this process on macOS, so I added some TFTP instructions for macOS.

Fohdeesha commented:
Thanks for the contribution! However I see there's also a change to fcx.md, for changes like this, it would need to be made to every individual switch page to stay uniform, and I think users on linux (or osx) probably already know how to connect to a serial device. If you remove the FCX changes I can go ahead and merge. Also, was it not required to disable the OSX firewall for this to work? I know it's typically pretty aggressive about blocking ports like these

evanreichard commented:
@Fohdeesha 

> However I see there's also a change to fcx.md, for changes like this, it would need to be made to every individual switch page to stay uniform, and I think users on linux (or osx) probably already know how to connect to a serial device

Totally fair. I removed the changes to that file.

> Also, was it not required to disable the OSX firewall for this to work? 

Yep! Forgot to note that. Added that to the tutorial as well.


____

Let me know if there's anything else you'd like changed!

Fohdeesha commented:
Sorry indeed work has been quite crazy lately. Thank you for the contribution! It's been merged and is now live on the site
